src/utils/enums.js

```diff
@@ -251,6 +266,26 @@ export const SUPPORTED_NETWORK_TYPES = {
     BSC_TESTNET_URL
   ),
   [XDAI]: createNetwork(XDAI, XDAI_DISPLAY_NAME, XDAI_CODE, XDAI_BLOCK_EXPLORER, XDAI_TICKER, 'xDai Network Token', 'xdai.svg', XDAI_URL),
+  [RSK_MAINNET]: createNetwork(
+    RSK_MAINNET,
+    RSK_MAINNET_DISPLAY_NAME,
+    RSK_MAINNET_CODE,
+    RSK_MAINNET_BLOCK_EXPLORER,
+    RSK_MAINNET_TICKER,
+    'RSK',
+    'rsk.png',
```

Comment:
we need to upload logos for rsk here:  'https://images.toruswallet.io', ideally in svg